code/network/multimsgs.cpp

```diff
@@ -8020,6 +8020,60 @@ void process_non_homing_fired_packet(ubyte* data, header* hinfo)
 	}
 }
 
+void send_animation_triggered_packet(int animationId, int pmi, const animation::ModelAnimationDirection& direction, bool force, bool instant, const int* time) {
```

Comment:
I don't see the corresponding changes in multi_obj.cpp . Did we already do these?